api/search.js

```diff
@@ -53,14 +53,31 @@ router.post('/recipes', asyncHandler(async (request, response, _next) => {
     });
   }
 
-  body.query.bool.filter = {
-    range: {
-      total_time: {
-        gte: fromCookTime,
-        lte: toCookTime,
+  body.query.bool.filter = [
+    {
+      bool: {
+        should: [
+          {
+            range: {
+              total_time: {
+                gte: fromCookTime,
+                lte: toCookTime,
+              },
+            },
+          },
+          {
+            bool: {
+              must_not: {
+                exists: {
+                  field: 'total_time',
```

Comment:
@guyschwartzberg-student This will currently give you results without cooking time even though you've explicitly requested cooking time

Reply:
When I tried to acquire recipes using your implementation it didn't show me recipes without cooking time - So how are you supposed to "not request" cooking time?